.gitlab-ci.yml

```diff
@@ -66,7 +66,7 @@ flake8-diff:
   image: pipelinecomponents/flake8:f087c4c
   script:
     - apk add --update --no-cache git
-    - git diff -U0 origin/master | flake8 --verbose --diff
+    -  git diff -U0 origin/master -- . ':(exclude)framework/tlvf/tlvf.py' | flake8 --verbose --diff
```

Comment:
#1249 removes flake8-diff entirely instead, which IMO is better :-)

Reply:
Done